README.md

```diff
@@ -121,6 +126,59 @@ kubectl rook-ceph ceph versions
 }
 ```
 
+### Debug Mode
+
+Debug mode can be useful when a mon's or OSD needs advanced maintenance operations that require the daemon to be stopped. Ceph tools such as `ceph-objectstore-tool`,`ceph-bluestore-tool`, or `ceph-monstore-tool` are commonly used in these scenarios. Debug mode will set up the mon or OSD so that these commands can be run.
+
+Debug mode will automate the following:
+1. Scale down the existing mon or OSD deployment
+2. Start a new debug deployment where operations can be performed directly against the mon or OSD without that daemon running
+   
+   a. The main container sleeps so you can connect and run the ceph commands
+   
+   b. Liveness and startup probes are removed
+
+   c. If alternate Image is passed by --alternate-image flag then the new debug deployment container will be using alternate Image.
+
+Debug Mode will work like it will scale down the existing deployment, start a new debug deployment without probes and that container pod will sleeps forever so you can connect and run ceph tools commands.
+
+For example, start the debug pod for mon `b`:
+```console
+kubectl rook-ceph debug start rook-ceph-mon-b
+```
+```text
+setting debug mode for "rook-ceph-mon-b"
+setting debug command to main container
+deployment.apps/rook-ceph-mon-b scaled
+deployment.apps/rook-ceph-mon-b-debug created
+```
+
+Now connect to the daemon pod and perform operations:
+```console
+kubectl exec <debug-pod> -- <ceph command>
+```
+
+You can also use alternate-image while starting the debug mode:
```

Comment:
I'd suggest removing lines 161-164, the alternate-image flag seems self explanatory. Let's keep this example focused.